website/src/pages/download.js

```diff
@@ -49,6 +49,50 @@ function Download() {
+                                >
+                                    <div className="panel--icon">
+                                        <i className="feather icon-download"></i>
+                                    </div>
+
+                                    <div className="panel--title">0.7.1</div>
+
+                                    <a href="https://www.apache.org/dyn/closer.lua/incubator/pinot/apache-pinot-incubating-0.7.1/apache-pinot-incubating-0.7.1-src.tar.gz">
+                                        <div className="panel--title">
+                                            Official source release
+                                        </div>
+                                    </a>
+                                    <a href="https://downloads.apache.org/incubator/pinot/apache-pinot-incubating-0.7.1/apache-pinot-incubating-0.7.1-src.tar.gz.sha512">
+                                        <div className="panel--subtitle">
+                                            SHA512
+                                        </div>
+                                    </a>
+                                    <a href="https://downloads.apache.org/incubator/pinot/apache-pinot-incubating-0.7.1/apache-pinot-incubating-0.7.1-src.tar.gz.asc">
+                                        <div className="panel--subtitle">
+                                            ASC
+                                        </div>
+                                    </a>
+
+                                    <a href="https://www.apache.org/dyn/closer.lua/incubator/pinot/apache-pinot-incubating-0.7.1/apache-pinot-incubating-0.7.1-bin.tar.gz">
+                                        <div className="panel--title">
+                                            Official binary release
+                                        </div>
+                                    </a>
+                                    <a href="https://downloads.apache.org/incubator/pinot/apache-pinot-incubating-0.7.1/apache-pinot-incubating-0.7.1-bin.tar.gz.sha512">
+                                        <div className="panel--subtitle">
+                                            SHA512
+                                        </div>
+                                    </a>
+                                    <a href="https://downloads.apache.org/incubator/pinot/apache-pinot-incubating-0.7.1/apache-pinot-incubating-0.7.1-bin.tar.gz.asc">
+                                        <div className="panel--subtitle">
+                                            ASC{" "}
+                                        </div>
+                                    </a>
+                                </a>
+                            </div>
                             <div className="col">
                                 <a
                                     href="https://www.apache.org/dyn/closer.lua/incubator/pinot/apache-pinot-incubating-0.7.0/apache-pinot-incubating-0.7.0-bin.tar.gz"
```

Comment:
should we remove 0.7.0 release?

Reply:
done, then shall we merge 0.7.0 release notes to 0.7.1 as well or just keep it separated?